src/modules/infobox/components/ImportContent.vue

```diff
@@ -0,0 +1,254 @@
+<template>
+    <div data-infobox="height-reference" class="import-overlay-content">
+        <div class="input-group d-flex">
+            <input
+                type="text"
+                class="form-control search-bar-input"
+                :placeholder="$t('import_online_placeholder')"
+                :value="importValue"
+                data-cy="import"
+                @input="onInputChange"
+                @focus="showProviders"
+                @keydown.down.prevent="goToFirstResult"
+                @keydown.esc.prevent="hideProviders"
+            />
+            <div v-if="listShown" ref="providers" class="providers-list-container">
+                <div class="providers-list">
+                    <div
+                        v-for="(provider, key) in filteredList"
+                        :key="provider"
+                        :tabindex="key"
+                        class="providers-list-item"
+                        @keydown.up.prevent="() => goToPrevious(key)"
+                        @keydown.down.prevent="() => goToNext(key)"
+                        @keydown.home.prevent="goToFirst"
+                        @keydown.end.prevent="goToLast"
+                    >
+                        <span
+                            v-html="provider.toShow"
+                            @click="() => onChooseProvider(provider.value)"
```

Comment:
The `click` event should be on the parent `div` otherwise when click on the left of the text nothing happens even if the row is selected and hovered

Reply:
updated